res/controllers/common-controller-scripts.js

```diff
@@ -368,6 +368,7 @@ bpm.tapButton = function(deck) {
 // ----------------- Common regular expressions --------------------------
 script.samplerRegEx = /\[Sampler(\d+)\]/ ;
 script.channelRegEx = /\[Channel(\d+)\]/ ;
+script.channelRegExExact = /^\[Channel(\d+)\]$/ ;
```

Comment:
Hm, it seems a shame to have to duplicate the regexes just to add ^$. It looks like there isn't a JS helper for "match the whole string" without adding that to the regex? I guess you could do:

```javascript
var exactMatch = function(regex, string) {
  var match = string.match(regex);
  return match !== null && match.index === 0 && match.length === string.length;
}
```

> there may be a use case for matching the [ChannelX] part of EQ rack or QuickEffectRack groups.

Isn't that what eqRegEx or quickEffectRegEx would give you? (i.e. group.match(eqRegEx)[1] would give you the eq/quickeffect rack's group). 

(also maybe we should remove the square brackets from eqRegEx / quickEffectRegEx so that the user doesn't have to re-add them to the matched value to get the group)